wordpress-mobile/WordPress-iOS
# Added alertview to ask for confirmation before deleting user activity log https://github.com/wordpress-mobile/WordPress-iOS/issues/13774

Fixes #13774 
Added popup asking the user to confirm the deletion of old activity logs

To test:
Go to Account (Me) > Help & Support > Activity Logs > Tap on 'Clear old activity logs' and a popup appears asking the user to confirm deletion of old activity logs

![Simulator Screen Shot - iPhone Xʀ - 2020-04-02 at 14 15 12](https://user-images.githubusercontent.com/1892674/78261435-cfe59b80-7531-11ea-9eaa-147fddca67d7.png)

PR submission checklist:
